provider/resources.go

```diff
@@ -4724,6 +4725,8 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 			"aws_batch_compute_environment": {Tok: awsDataSource(batchMod, "getComputeEnvironment")},
 			"aws_batch_job_queue":           {Tok: awsDataSource(batchMod, "getJobQueue")},
 			"aws_batch_scheduling_policy":   {Tok: awsDataSource(batchMod, "getSchedulingPolicy")},
+			// chatbot
+			"aws_chatbot_slack_workspace": {Tok: awsDataSource(chatbotMod, "getSlackWorkspace")},
```

Comment:
I added the new resources to the separate plugin based resources, but it doesn't look like we have a separate place to put plugin based data sources.